boards/stm32h743zi/src/bin/telemetry.rs

```diff
@@ -0,0 +1,127 @@
+#![no_std]
+#![no_main]
+
+use defmt::*;
+use embassy_executor::Spawner;
+use embassy_net::{Ipv4Address, Ipv4Cidr, Stack, StackResources};
+use embassy_stm32::{
+    bind_interrupts,
+    eth::{self, generic_smi::GenericSMI, Ethernet, PacketQueue},
+    gpio::Pin,
+    peripherals::{self, ETH},
+    rng::{self, Rng},
+    time::Hertz,
+    Config,
+    rcc
+};
+use embassy_time::{Duration, Timer};
+use hyped_boards_stm32h743zi::{
+    log::log,
+    tasks::{
+        mqtt_send_and_recv::{mqtt_recv_task, mqtt_send_task, net_task},
+        test_mqtt_tasks::{button_task, five_seconds_task},
+    },
+    config::{BROKER_CIDR,GATEWAY_IP}
+};
+
+use hyped_core::log_types::LogLevel;
+use rand_core::RngCore;
+use static_cell::StaticCell;
+use {defmt_rtt as _, panic_probe as _};
+
+bind_interrupts!(struct Irqs {
+    ETH => eth::InterruptHandler;
+    RNG => rng::InterruptHandler<peripherals::RNG>;
+});
+
+#[embassy_executor::main]
+async fn main(spawner: Spawner) -> ! {
+    let mut config = Config::default();
+    {
+        use embassy_stm32::rcc::*;
+        config.rcc.hse = Some(Hse {
+            freq: Hertz(8_000_000),
+            mode: HseMode::Bypass,
+        });
+        config.rcc.pll1 = Some(rcc::Pll {
+            source: PllSource::HSE,
+            prediv: PllPreDiv::DIV4,
+            mul: PllMul::MUL216,
+            divp: Some(rcc::PllDiv::DIV2), // check me
+            divq: None,
+            divr: None,
+        });
+        config.rcc.ahb_pre = AHBPrescaler::DIV1;
+        config.rcc.apb1_pre = APBPrescaler::DIV4;
+        config.rcc.apb2_pre = APBPrescaler::DIV2;
+        config.rcc.apb3_pre = APBPrescaler::DIV2;
+        config.rcc.apb4_pre = APBPrescaler::DIV2;
+        config.rcc.sys = Sysclk::PLL1_P;
+    }
+    let p = embassy_stm32::init(config);
+
+    spawner.spawn(button_task(p.PC13.degrade())).unwrap();
+
+    log(LogLevel::Info, "Hello World!").await;
+
+    // let seed: u64 = 0xdeadbeef;
+    let mut rng = Rng::new(p.RNG, Irqs);
+    let mut seed = [0; 8];
+    rng.fill_bytes(&mut seed);
+    let seed = u64::from_le_bytes(seed);
+
+    let mac_addr: [u8; 6] = [0x00, 0x00, 0xDE, 0xAD, 0xBE, 0xEF];
+
+    static PACKETS: StaticCell<PacketQueue<4, 4>> = StaticCell::new();
+    let device = Ethernet::new(
+        PACKETS.init(PacketQueue::<4, 4>::new()),
+        p.ETH,
+        Irqs,
+        p.PA1,
+        p.PA2,
+        p.PC1,
+        p.PA7,
+        p.PC4,
+        p.PC5,
+        p.PG13,
+        p.PB13,
+        p.PG11,
+        GenericSMI::new(0),
+        mac_addr,
+    );
+
+    // let config = embassy_net::Config::dhcpv4(Default::default());
+    let config = embassy_net::Config::ipv4_static(embassy_net::StaticConfigV4 {
+        address: BROKER_CIDR,
```

Comment:
`BROKER_CIDR` I think is actually the static IP that the board will have, so should be called `BOARD_STATIC_IP` or something instead